src/Mod/PartDesign/Gui/TaskLoftParameters.cpp

```diff
@@ -282,6 +281,7 @@ void TaskLoftParameters::indexesMoved()
 
 void TaskLoftParameters::clearButtons() {
 
+    ui->buttonProfileBase->setChecked(false);

```

Comment:
Why this addition?

Reply:
Otherwise, if we press the profile button and change the profile, the button remains pressed. It also remains pressed when the add/remove section buttons are pressed.
![Screenshot at 2021-11-08 17-27-52](https://user-images.githubusercontent.com/4316933/140828257-567bd4d1-becd-49cf-954f-019515c65644.png)

Reply:
OK, I only don't understand yet why this was not necessary before your PR.

Reply:
It was. It was a minor fix so I forgot about it and left it in. It needs some more treatment similar to https://github.com/FreeCAD/FreeCAD/blob/3a5c5576ea269b8bae14325db844e4d9201a7fa4/src/Mod/Fem/Gui/TaskFemConstraintFixed.cpp#L249, because right now if you press, for example "remove" when "add" is pressed, "add" remains pressed. If you want I could make it a separate PR, though.

Reply:
> because right now if you press, for example "remove" when "add" is pressed, "add" remains pressed. If you want I could make it a separate PR, though.

I see. Yes please make a separate PR. Then I can backport this for 0.19.

Reply:
OK change removed from here to be put in a separate PR.

Reply:
> OK change removed from here to be put in a separate PR.

Thanks. Please make the PR soon, because to be able to backport easily, I need to put that new PR in before this PR can go in.

Reply:
@donovaly just a note : why require another PR? My own rule of thumb when I mix bug fix and feature enhancement is to have the bug fix in a separate commit (not a separate PR). Isn't that enough for backporting?